### PR TITLE
fix(ecs): enable filtering on container image and target group

### PR DESCRIPTION
**Pull request by pkandasamy91**
Enable filtering on container image and target group when creating or updating a server group.

Fixes: https://github.com/spinnaker/spinnaker/issues/5309

Screen Shot 2020-02-05 at 9 54 04 AM
Screen Shot 2020-02-05 at 9 54 13 AM

Screen Shot 2020-02-05 at 9 54 32 AM

Screen Shot 2020-02-05 at 9 55 07 AM

### CODE REVIEW

**Comment by clareliguori** [comment]
Can you add a screenshot?

**Comment by allisaurus** [comment]
Can you edit the commit msg suffix to `fix(ecs)`? See [previous](https://github.com/spinnaker/deck/pull/6746) PRs.

**Comment by pkandasamy91** [comment]
> Can you add a screenshot?

Added

**Comment by pkandasamy91** [comment]
> Can you edit the commit msg suffix to `fix(ecs)`? See [previous](https://github.com/spinnaker/deck/pull/6746) PRs.

Amended

File: app/scripts/modules/ecs/src/serverGroup/configure/wizard/container/Container.tsx [hunk 1]
```diff
@@ -165,27 +161,21 @@ export class Container extends React.Component<IContainerProps, IContainerState>
...
     });
 
     const targetGroupInputs = this.state.targetGroupMappings.map(function(mapping, index) {
       return (
         <tr key={index}>
           <td>
-            <select
-              className="form-control input-sm"
+            <TetheredSelect
+              placeholder="Select an target group to use..."
```
Comment (clareliguori) [inline app/scripts/modules/ecs/src/serverGroup/configure/wizard/container/Container.tsx]: an -> a

File: app/scripts/modules/ecs/src/serverGroup/configure/wizard/taskDefinition/TaskDefinition.tsx [hunk 2]
```diff
@@ -289,15 +282,13 @@ export class TaskDefinition extends React.Component<ITaskDefinitionProps, ITaskD
             />
           </td>
           <td>
-            <select
-              className="form-control input-sm"
+            <TetheredSelect
+              placeholder="Select an target group to use..."
```
Comment (clareliguori) [inline app/scripts/modules/ecs/src/serverGroup/configure/wizard/taskDefinition/TaskDefinition.tsx]: Same, an -> a

**Pull request merged**